021-IdentityForApps/README.md

```diff
@@ -1,8 +1,112 @@
-# Welcome to the Identity for Applications Hack!
+# Identity for Developers
 
-This is a challenge-based hack using scenarios dealing with identity using Azure AD from an application development perspective.
+## Introduction
 
-## **This is a work in progress**
+The Identity for Developers Hack will provide you a deep dive experience into enabling customer-facing identity solutions for your applications. Identity is a booming area of the Microsoft Cloud platform and enabling identity solutions in your engagements allows for faster production deployments. Azure Active Directory B2C enables you to provide custom identity access management solutions for your applications.
 
-We're working hard on this hack and it will be released soon!
 
+![B2C Overview](https://docs.microsoft.com/en-us/azure/active-directory-b2c/media/overview/scenario-singlesignon.png)
+## Learning Objectives
+
+In this hack, we'll enable identity solutions using Azure Active Directory B2C, demonstrating how to:
+
+  1. Create and integrate out of the box user flows
+
+  2. Create and integrate customized policies
+
+  3. Call REST APIs for token enrichment and claims validation
+
+  4. Monitor and report on your B2C tenant
+
+## Overall Architecture
+
+In this hack, you will build Azure AD B2C policies that enable users to be able to sign up, sign in, edit their profile, and delete their account. Along the way, you will enable API integration, conditional access checks, monitoring, and other services. The below diagram shows the various services this hack involves, along with the corresponding challenges where you will encounter these services. The section that follows this provides additional information and links to the specific challenges.
+
+![Contoso Manufacturing Consultant App Architecture](./Images/Azure_AD_B2C_WTH_Final_Design.png)
+## Challenges
+
+Challenge 0: **[Prepare your workstation for Identity development](Student/00-pre-reqs.md)**
```

Comment:
Can you go through all your student and coach challenge guides and make sure the titles match what's here in the table of contents.... for example:

ToC - Challenge 1: Provision an Azure AD B2C Tenant
Challenge Text - Challenge 1: Getting Started with B2C

They should be consistent, but it's your choice which one is most appropriate.

Reply:
updated coach and student challenge files and readmes to use same titles